frontend/src/types.ts

```diff
@@ -2097,15 +2097,16 @@ export enum SurveyType {
 export interface SurveyAppearance {
     backgroundColor?: string
     submitButtonColor?: string
-    textColor?: string
```

Comment:
I think the other PR in the site app still supports `textColor` and `descriptionTextColor`, we should remove it there too then